backend/business-partner-agent/src/main/java/org/hyperledger/bpa/impl/PartnerManager.java

```diff
@@ -193,4 +198,11 @@ public void acceptPartner(@NonNull UUID partnerId) {
         cm.acceptConnection(connectionId);
     }
 
+    public void sendMessage(@NonNull UUID id, String content) {
+        final Optional<Partner> dbP = repo.findById(id);
+        if (dbP.isPresent()) {
```

Comment:
You have to check two things here. 1. If the connection id is set, as it might be null in some states 2. If the partner has ariesSupport== true as we have none aries partners in web mode.

Reply:
thanks.  will put those checks in.